recipe/meta.yaml

```diff
@@ -22,14 +21,15 @@ requirements:
   host:
     - python
     - pip
-    - pyct >=0.4.4
-    - param >=1.9.3
-    - setuptools >=30.3.0
+    - pyct
+    - param <2.0
```

Comment:
is this contrain required?

Reply:
The conda-forge uses the upper bound. But I don't see that param project is developing v2.*.*
I'll remove it